sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -573,6 +573,13 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ENABLE_TYPED_IMPERATIVE_AGGREGATE = conf("spark.rapids.sql.typedImperativeAggregate.enabled")
```

Comment:
This setting was added before the introduction of "associated fallback". With this setting, we can disable TypedImperativeAggregate by default, to get rid of unexpected crash with default configuration.
Perhaps we can remove this option, since we rewrite all partial-accelerated cases in current.

Reply:
Ok, if you know that the aggregate does the right thing in all cases and won't crash then I would remove the extra settings.

Reply:
I've removed the setting.